basics/hello_world.tex

```diff
@@ -39,10 +39,7 @@
 \textbf{Wie gehe ich dabei vor?}
 
 Um den Programmcode in eine Textdatei zu schreiben verwenden wir einen Texteditor.
-Dies kann mit fast jedem einfachen Texteditor bewerkstelligt werden. Wir verwenden beispielsweise im Vokurs gedit unter Ubuntu.
-Unter Windows kannst du den Editor oder notepad++ verwenden, unter MacOS Textedit. 
-In der fortgeschrittenen Programmierung werden intelligente Programmierumgebungen verwendet, die jedoch in diesem Kurs vernachlässigt werden.
-
+Dies kann mit fast jedem einfachen Texteditor bewerkstelligt werden. Populäre konsolenbasierte Editoren sind \texttt{vim} und \texttt{nano}. Es existieren aber auch grafische Editoren, die speziell für das Schreiben von Programmcode entwickelt wurden, wie \texttt{Visual Studio Code} oder \texttt{CLion}.
```

Comment:
Wir sind noch am überlegen ob wir den Hinweis, dass JetBrains free ist ins Skript packen oder nur mündlich weitergeben.

Meinungen?

Reply:
jet brains ist toll aber nicht für erstes, VScode ist deshalb gementioned weil es komplett plug and play ist. Auf CLion kann man hinweisen